javascript/javascript1/week1/Goodboy-Oldboy.js

```diff
@@ -0,0 +1,13 @@
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2037;
+const humanYear = dogYearFuture - dogYearOfBirth;
+const dogYear = 7 * humanYear;
+
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears === true) {
```

Comment:
When you check if something is `true` you can simply write it like this:
```suggestion
if (shouldShowResultInDogYears) {
```